### ISSUE TITLE
Examples/ROS/ORB_SLAM3 Path

### ISSUE THREAD

**Issue by HVKHVK** [UZ-SLAMLab/ORB_SLAM3#454]
Examples/ROS/ORB_SLAM3 path changed to Examples_old/ROS/ORB_SLAM3. Update build_ros.sh

**Comment by DavideLuc** [comment]
Is ROS working with the new YAML configuration? For me didn't work with stereo configuration, it's asking for old parameters on the YAML file.

**Comment by HVKHVK** [comment]
Not really. I was not able to compile the ROS part completely while trying to overcome the issue. I am tracking several other issues and I will wait for more stable library. Till then ORB_SLAM2.

**Comment by nindanaoto** [comment]
@HVKHVK You can use the code for this PR for a compilable ROS example.

**Comment by robin-shaun** [comment]
I use the PR #447, for pinhole cameras, it's OK. But for fisheye cameras such as T265.  ERROR: Calibration parameters to rectify stereo are missing
